source/common/listener_manager/filter_chain_manager_impl.cc

```diff
@@ -28,7 +28,7 @@ namespace FilterChain {
 // when no IP matcher is configured.
 Network::Address::InstanceConstSharedPtr fakeAddress() {
   CONSTRUCT_ON_FIRST_USE(Network::Address::InstanceConstSharedPtr,
-                         Network::Utility::parseInternetAddress("255.255.255.255"));
+                         Network::Utility::parseInternetAddressNoThrow("255.255.255.255"));
```

Comment:
Is this the only (non-test) change which actually eliminates a potential throw? It looks like all the other call sites switch from the throwing version to NoThrow but then throw if null.